### PR TITLE
IN week days with BE and FE for /dashboard 

### PR DESCRIPTION
**Pull request by g4rry420**
Signed-off-by: gurkiran_singh <gurkiransinghk@gmail.com>

### What this PR does (required):
- User to select availability based on week days

### Screenshots / Videos (required):
Capture

### Any information needed to test this feature (required):
- Create dog sitter account using `become a sitter` link and filled the required info.
- Login using non dog sitter account and test the functionality.

### Any issues with the current functionality (optional):
No

### CODE REVIEW

File: client/src/components/EditProfile/EditProfile.tsx [hunk 1]
```diff
@@ -125,22 +147,20 @@ const EditProfile: FC<Props> = (props) => {
...
             phone: Yup.string()
               .required('Please, enter your phone number')
               .matches(/^[0-9]+$/, 'Must contain numbers only')
               .length(10, 'Number must be 10 digits'),
             priceRate: loggedInUser?.isDogSitter
               ? Yup.string()
-                  .required('Please, enter your phone number')
+                  .required('Please, enter your Price Rate')
                   .matches(/^[0-9]+$/, 'Must contain numbers only')
                   .max(3, 'Value must br less than $100')
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx]:
```suggestion
                  .max(2, 'Value must be less than $100')

```

Comment (moffatethan) [reply client/src/components/EditProfile/EditProfile.tsx]: Also, wouldn't this still allow $300 or $999

Comment (g4rry420) [reply client/src/components/EditProfile/EditProfile.tsx]: Yes, I change max to 2.

**Review by joelmackenz — APPROVED** [review]
I can see why the main branch wasn't working -- all related to the availability days. This looks good to me. "Yup" looks like an interesting dependency.

**Pull request merged**